eng/pipelines/apiview-review-gen-cadl.yml

```diff
@@ -45,7 +45,7 @@ jobs:
 
   - pwsh: |
       npm install -g @cadl-lang/compiler@0.37.0
-      npm install -g @azure-tools/cadl-apiview@0.3.0
+      npm install -g @azure-tools/cadl-apiview@latest

```

Comment:
This will probably fail since the latest version of cadl-apiview uses a different version of cadl compiler. It might be sufficient to just install cadl-apiview, which should pull in the supported version of the compiler.

Reply:
Just as an FYI @ruowan is working on putting this version as a global setting inside of the specs repo. Once that gets worked out we should try to figure out how to use that same version in this pipeline.

Reply:
Current in PR pipeline, I just use the latest version. Once the global config is done in spec repo, I will make some changes to pick up that global version.